content/blog/2022/learn-nextflow-in-2022.md

```diff
@@ -0,0 +1,182 @@
+title=Learn Nextflow in 2022
+date=2022-01-21
+type=post
+description=Resources for learning Nextflow: 2022 Revision
```

Comment:
It was nicer "Learning Nextflow in 2022"! 😄

Reply:
Updated.